src/utils.ts

```diff
@@ -44,4 +44,57 @@ export class NullOutstreamStringWritable extends stream.Writable {
             callback();
         }
     }
-};
\ No newline at end of file
+};
+
+export const checkIfEnvironmentVariableIsOmitted = (key: string): boolean => {
+
+    const omitEnvironmentVariables: string [] = [

```

Comment:
How have you reached at this and the below list?

Reply:
By comparing the list of environment variables on the host and container side. These are the ones for packages/path installed on the host or one which conflict with the container